share/completions/connmanctl.fish

```diff
@@ -0,0 +1,40 @@
+function __fish_print_connman_services
+    connmanctl services | string replace -r '.* (.*)' '$1'
```

Comment:
If this is where you'd like the description to be generated, you could do `string replace -r '(.*) (.*)' '$2\t$1'`.